docs/js/doc-page.js

```diff
@@ -60,6 +188,19 @@ export default ({ data, location }) => {
         {metaDescription && (
           <meta name="description" content={metaDescription} />
         )}
+        <script type="application/ld+json">
+          {JSON.stringify(
+            getDocPageSchema({
+              isLatest: slug.includes('docs/latest'),
```

Comment:
Why are we calling out whether the page is "latest" here? Does it matter that pages for the 1.5 version are actually "latest" (the content is identical) but their slugs don't include 'docs/latest' (they include 'docs.1.5' instead)?

Reply:
This is merely for defining the `@id` of the first breadcrumb item in the Schema definition, which is the main page for the current docs. As the website has both `/docs/latest` and `/docs/1.5` live, it's important to make this distinction to prevent mixing both.

That said, we can get the main page's URL through the current page's slug, with something like:

`relativeToAbsUrl(slug.match(/\/docs\/[\d\.(latest)]*/)[0])`